src/Internal/Serialization.purs

```diff
@@ -19,6 +19,7 @@ module Ctl.Internal.Serialization
   , publicKeyFromPrivateKey
   , privateKeyFromBytes
   , makeVkeywitness
+  , newBigNum
```

Comment:
We should remove `newBigNum` from serialization - nothing uses it. Replace your uses in `Test.Serialization` with `Types.BigNum.fromString`